app/src/features/settings/components/BillingTeam/components/BillingDetails/components/ReviewJoinRequestModal/ReviewJoinRequestModal.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useCallback, useEffect, useState } from "react";
```

Comment:
Keep this in `BillingDetails/modals/ReviewJoinRequestModal/ReviewJoinRequestModal.tsx`